check_nextcloud.php

```diff
@@ -6,12 +6,13 @@
  * Monitoring plugin to check the status of nextcloud serverinfo app
  *
  * Copyright (c) 2019 Kevin Köllmann <mail@kevinkoellmann.de>
- *
+ * contribution: Conrad Beckert <kontakt@miradata.de>
  *
  * Usage: /usr/bin/php ./check_nextcloud.php -H cloud.example.com -u /ocs/v2.php/apps/serverinfo/api/v1/info
  *
  *
  * For more information visit https://github.com/koelle25/check_nextcloud
+ * For more information visit https://github.com/beccon4/check_nextcloud
```

Comment:
This line is also not needed (see above)

Reply:
ok